desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -38,11 +38,7 @@ TabDiveInformation::TabDiveInformation(QWidget *parent) : TabBase(parent), ui(ne
 	connect(ui->diveType, SIGNAL(currentIndexChanged(int)), this, SLOT(diveModeChanged(int)));
 	QString CSSSetSmallLabel = "QLabel { color: mediumblue; font-size: " +                           // Using label height
 		QString::number((int)(0.5 + ui->diveHeadingLabel->geometry().height() * 0.66)) + "px;}"; // .. set CSS font size of star widget subscripts
-#if defined(Q_OS_WIN)
```

Comment:
so if the suspicion was that this check is inverted (i.e., that the other style sheet is needed on Windows), have you given that a try (err, do you have access to a Windows system to try it)?
I need to start up my Windows VM, deal with an hour of mandatory updates and then see if one loops better than the other before merging this (yes, I realize that this wasn't used on Windows in the current code - I'm wondering if this would be a net improvement if it WAS used)